packages/frontend2/src/app/(new)/(other)/scaling/tvl/_components/scaling-tvl-chart.tsx

```diff
@@ -0,0 +1,274 @@
+'use client'
+
+import { type Milestone } from '@l2beat/config'
+import { useMemo } from 'react'
+import {
+  useScalingFilter,
+  useScalingFilterValues,
+} from '~/app/(new)/(other)/_components/scaling-filter-context'
+import { ChartTimeRangeControls } from '~/app/_components/chart/controls/chart-time-range-controls'
+import { Chart } from '~/app/_components/chart/core/chart'
+import { useChartLoading } from '~/app/_components/chart/core/chart-loading-context'
+import { ChartProvider } from '~/app/_components/chart/core/chart-provider'
+import { mapMilestones } from '~/app/_components/chart/utils/map-milestones'
+import { PercentChange } from '~/app/_components/percent-change'
+import { Skeleton } from '~/app/_components/skeleton'
+import { INFINITY } from '~/consts/characters'
+import { useCookieState } from '~/hooks/use-cookie-state'
+import { useLocalStorage } from '~/hooks/use-local-storage'
+import { type ScalingTvlEntry } from '~/server/features/scaling/tvl/get-scaling-tvl-entries'
+import { type TvlLayer2ProjectFilter } from '~/server/features/scaling/tvl/utils/project-filter-utils'
+import { type TvlChartRange } from '~/server/features/scaling/tvl/utils/range'
+import { api } from '~/trpc/react'
+import { cn } from '~/utils/cn'
+import { formatTimestamp } from '~/utils/dates'
+import { formatCurrency } from '~/utils/format'
+import { formatNumber } from '~/utils/format-number'
+import { UnitAndScaleControls } from '../../_components/unit-and-scale-controls'
+import { tvlRangeToReadable } from '../../_utils/tvl-range-to-readable'
+
+interface Props {
+  milestones: Milestone[]
+  tag?: string
+  entries: ScalingTvlEntry[]
+}
+
+export function SummaryTvlChart({ milestones, entries, tag = 'tvl' }: Props) {
```

Comment:
I would vote for changing the name from `SummaryTvlChart` to `StackedTvlChart`. Then the Summary chart would be simply TvlChart.

Reply:
Good idea, changing :pray: